packages/generators/generators/lib/templates/ts/plugin-package.json.hbs

```diff
@@ -7,7 +7,9 @@
     "description": "Description of {{ pluginName }} plugin",
     "kind": "plugin"
   },
-  "dependencies": {},
+  "dependencies": {
+    "@strapi/icons": "^1.3.1"
```

Comment:
@joshuaellis We should bump this to 1.4.0, which is the latest version used in core/admin.

Reply:
Good question, I think there a couple of options here:

* We're using a caret so this will use the most up to date version (excluding a major v e.g. `2.0.0`) so I think it might be okay to leave?
* We could use an asterisk because we _know_ the DS version will be enforced by the admin panel, so it might be better they use the most up to date version?
* We could just supply it as a peer dep as we know it'll be installed by the admin panel, but for local development it might be needed to include it in the dev deps.

My with trying to keep it in lock-step with the admin panel is yeah, someone needs to remember to manually update this one file which could be problematic and we don't want _another_ script to use?

CC @alexandrebodin be great to get your opinion on what you think would be best moving forward?